doc/man3/RAND_DRBG_get0_master.pod

```diff
@@ -0,0 +1,81 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_get0_master,
+RAND_DRBG_get0_public,
+RAND_DRBG_get0_private
+- get access to the global RAND_DRBG instances
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ RAND_DRBG *RAND_DRBG_get0_master(void);
+ RAND_DRBG *RAND_DRBG_get0_public(void);
+ RAND_DRBG *RAND_DRBG_get0_private(void);
+
+
+=head1 DESCRIPTION
+
+These functions are part of the OpenSSL L<RAND_DRBG(7)> API.
+
+The default RAND API implementation (RAND_OpenSSL()) utilizes three
+shared DRBG instances which are accessed via the RAND API:
+
+The <public> and <private> DRBG are thread-local instances, which are used
+by RAND_bytes() and RAND_priv_bytes(), respectively.
+The <master> DRBG is a global instance, which is not intended to be used
```

Comment:
What is the rationale for exporting something _not intended to be used_?

Reply:
I meant only to say that it is not intended to be used for generating random bytes directly. Of course there are uses for obtaining a pointer to the master DRBG. For example, if you want to exchange the reseeding callbacks entirely. I will think about it and reformulate this paragraph.

Reply:
> For example, if you want to exchange the reseeding callbacks entirely.

... or if you want to chain your own DRBG instances to the master DRBG.